src/controllers/AuthController.js

```diff
@@ -31,6 +36,63 @@ const forgotPassword = async (request, response) => {
   }
 };
 
-export default {
-  forgotPassword
+/**
+ * Update user verified status
+ *
+ * @param {object} req
+ * @param {object} res
+ * @returns {json} - json
+ */
+
+const updateStatus = async (req, res) => {
+  const secret = process.env.ACCOUNT_VERIFICATION_SECRET;
+  const { params: { token } } = req;
+
+  const tokenValue = await jwt.verify(token, secret, (err, decoded) => {
```

Comment:
Kindly consider extracting this to a helper function for verifying JWT. It will come in handy in features being worked on and other PRs that are currently up.

Reply:
done